src/scheduler/sched_ifl_wrappers.cpp

```diff
@@ -153,7 +177,11 @@ send_attr_updates(int virtual_sd, resource_resv *resresv, struct attrl *pattr)
 preempt_job_info *
 send_preempt_jobs(int virtual_sd, char **preempt_jobs_list)
 {
-    return pbs_preempt_jobs(virtual_sd, preempt_jobs_list);
+	preempt_job_info *ret;
+
+    ret = pbs_preempt_jobs(virtual_sd, preempt_jobs_list);
```

Comment:
As of today, pbs_preempt_jobs already detects if it is not able to read data from one of the servers it clears the responses from other servers too. Are you going to address that in later PRs?

Reply:
Thanks for pointing this out, I hadn't thought about it, yes, it'll need to be updated.